src/containers/tutor-profile/profile-info/ProfileContainerMobile.jsx

```diff
@@ -6,30 +6,28 @@ import TitleWithDescription from '~/components/title-with-description/TitleWithD
 import AppChipList from '~/components/app-chips-list/AppChipList'
 import ProfileDoneItemsList from '~/components/icon-with-text-list/ProfileDoneItemsList'
 
-import img from '~/assets/img/tutor-profile-page/avatar.png'
 import { styles } from '~/containers/tutor-profile/profile-info/ProfileInfo.styles'
 
 const ProfileContainerMobile = ({
   actionIcon,
   accInfo,
   buttonGroup,
   defaultQuantity,
-  subjectChips,
-  doneItems
+  doneItems,
+  userData
 }) => {
+  const subjectData = userData.mainSubjects.tutor.map((item) => item.name)
   return (
     <Box sx={styles.container}>
       <Box sx={{ display: 'flex', gap: '10px' }}>
         <Box sx={{ flex: 1 }}>
-          <Box component='img' src={img} sx={styles.img} />
+          <Box component='img' src={userData.photo} sx={styles.img} />
```

Comment:
if user doesn't have photo it looks horrible, u should fix it

Reply:
I guess, you can use  Avatar from MUI

Reply:
Fixed, Avatar added. Please review the updated screenshots.